src/components/accordion/Accordion.spec.jsx

```diff
@@ -163,4 +163,28 @@ describe('<Accordion>', () => {
     // hostNodes returns html elements and skip react components
     expect(accordion.find('[aria-expanded=true]').hostNodes()).toHaveLength(2);
   });
+
+  it('renders span with link styles when title is string', () => {
```

Comment:
`span` in this context is saying more than test does, maybe we can rephrase it?

Reply:
https://github.com/brainly/style-guide/pull/2052/commits/41349ae98be711232f036fdd6fbdbe915bdbf91a